### PR TITLE
Add Asset Modal

### PR DESCRIPTION
**Pull request by mathieu-lessard**
JIRA: [DXI-22446](https://jira.corp.adobe.com/browse/DXI-22446)

Test URLs:

- Before: https://main--adobe-gmo--hlxsites.hlx.live/
- After: https://chdemo--adobe-gmo--hlxsites.hlx.page/qa/assets

### CODE REVIEW

File: .github/workflows/playwright.yml.disabled [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Does not work, because Adobe flags the source IP as untrusted, and requires confirming a code sent by email.
+
+name: Playwright Tests
```
Comment (shiv-gup) [inline .github/workflows/playwright.yml.disabled:3]: Is this workflow needed? I see this workflow will only run when we push any code to `main` branch.

**Review by shiv-gup — APPROVED** [review]

File: .github/workflows/playwright.yml.disabled (hunk 1 above)
Comment (mathieu-lessard) [reply .github/workflows/playwright.yml.disabled:3]: @shiv-gup 

I believe this entire file is disabled due to the name being changed to .yml.disabled. 

Definitely calls into question the usefulness of the entire file and the e2e file, but I don't believe this should cause an issue in the main branch.

Comment (mathieu-lessard) [reply .github/workflows/playwright.yml.disabled:3]: Discussed in slack. Agreed to leave as is for now as it is disabled and understand their purpose later.

**Review by cheintzman — APPROVED** [review]
Tested locally. LGTM!

**Pull request merged**